packages/toolkit/scripts/test/launchTestUtilities.ts

```diff
@@ -18,7 +18,7 @@ const minimum = 'minimum'
 
 const disableWorkspaceTrust = '--disable-workspace-trust'
 
-type SuiteName = 'integration' | 'e2e' | 'unit'
+type SuiteName = 'integration' | 'e2e' | 'unit' | 'web'
```

Comment:
should we be consistence with `browser` vs `web`? applicable throughout cr

Reply:
Justin recently mentioned we should start using `web` instead since that seems to be the branding VS Code is going with. So we'll be using `web` for now and swapping uses of `browser`